src/main/java/com/github/valfirst/slf4jtest/LoggingEvent.java

```diff
@@ -49,7 +52,7 @@ public class LoggingEvent {
     private final ImmutableList<Object> arguments;
 
     private final Optional<TestLogger> creatingLogger;
-    private final Instant timestamp = new Instant();
+    private final Instant timestamp = Instant.now();
```

Comment:
It would be nice if this could take a `Clock`, but I don't see a good way to do that without adding more complexity in API or state, or making the tests less direct.